corehq/apps/app_manager/migrations/0001_initial.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+from django.conf import settings
+from django.core.management import call_command
+
+from dimagi.utils.couch import sync_docs
+
+import corehq.apps.app_manager.models.common as models_common
+import corehq.apps.app_manager.models.schedules as models_schedules
+
+def sync_app_manager_docs(apps, schema_editor):
+    sync_docs.sync(models_common, verbosity=2)
+    sync_docs.sync(models_schedules, verbosity=2)
+
+
+class Migration(migrations.Migration):
+
+    dependencies = []
+
+    operations = [
+        migrations.RunPython(sync_app_manager_docs),
+    ]
```

Comment:
@dannyroberts So, I wrote this migration as an experiment and it caused more tests to pass so I committed it...but there are still test failures relating to the code @snopoke linked to. Your concerns make sense, think I'll be able to respond to them better once I get the current test failure figured out (and hopefully that response will be "fixed the underlying problem, deleted the migration").